students/Briadean/lesson02/grid_printer.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+"""Print an exact replica of the example grid"""
+def grid_printer_1():
+    row = "+" + " - " * 4 + "+" + " - " * 4 + "+"
+    column = "|" + " " * 12 + "|" + " " * 12 + "|"
+
+    print(row)
+    for x in range(4):
+        print(column)
+    print(row)
+    for x in range(4):
+        print(column)
```

Comment:
Nice loops! you can cut down on the number of "for..." statements by putting those loops you already have within a loop